assets/js/invenio_communities/community/communities-carousel/overrides/CarouselItem.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2016-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import _truncate from "lodash/truncate";
+import { Image } from "react-invenio-forms";
+import { Grid, Header, Item } from "semantic-ui-react";
+
+export const overriddenCarouselItem = ({community, defaultLogo, className}) => {
```

Comment:
```suggestion
export const CdsCarouselItem = ({community, defaultLogo, className}) => {
```
classes' names should be starting with a capital letter